frb_macros/src/lib.rs

```diff
@@ -10,25 +10,48 @@ use proc_macro::*;
 // frb-coverage:ignore-start
 #[proc_macro_attribute]
 pub fn frb(attribute: TokenStream, item: TokenStream) -> TokenStream {
```

Comment:
nit: would be great to have some unit tests to clearly show what this proc macro will do (show inputs, expected output, etc)

Reply:
Interesting. Do you think tests in the macro crate could work?

Reply:
IMHO yes, just standard rust unit tests, and consider `frb` as a normal rust function.
If not working because of `proc_macro_attribute`, maybe do something like `fn frb_inner` and test on that.

Our CI infra should already consider tests in frb_macros. To test this, we can firstly create a definitely failing test and see whether CI realize that.

Reply:
I'm hitting the "procedural macro API is used outside of a procedural macro" so probably might have to refactor the logic out to a different crate over the weekend.

Reply:
Hmm then maybe no need for this test, since we have abundant e2e tests.